pyproject.toml

```diff
@@ -68,7 +68,26 @@ target-version = "py39"
 
 [tool.ruff.lint]
 select = ["ALL"]
-ignore = ["D"]  #  comments
+ignore = [
```

Comment:
Are all these ignores needed? Codebase passes with just D ignored, is this for ruff format?